backend/app/adapter/graphql/resolver/changelog.go

```diff
@@ -0,0 +1,33 @@
+package resolver
+
+import (
+	"time"
+
+	"github.com/short-d/short/app/adapter/graphql/scalar"
+	"github.com/short-d/short/app/entity"
+)
+
+// ChangeLog represents ChangeLog entity and user Last Viewed ChangeLog time
+type ChangeLog struct {
+	changeLog    []Change
+	lastViewedAt time.Time
+}
+
+// Changes retrieves full ChangeLog
+func (c ChangeLog) Changes() []Change {
+	return c.changeLog
+}
+
+// LastViewedAt retrieves lastViewedAt for given user
+func (c ChangeLog) LastViewedAt() *scalar.Time {
+	return &scalar.Time{Time: c.lastViewedAt}
```

Comment:
```suggestion
        if c.lastViewedAt == nil {
		return nil
	}
	return &scalar.Time{Time: c.lastViewedAt}
```

Reply:
c.lastViewedAt cannot be nil it is not a pointer